content/en/deployment/lambda/index.md

```diff
@@ -0,0 +1,326 @@
+---
+title: AWS Lambda Deployment Overview
+linktitle: AWS Lambda Overview
+weight: 1
+description: >
+ Learn what an Armory CD-as-a-Service deployment to AWS Lambda is, how CD-as-a-Service connects to your AWS Account, and how deployment works.
+---
+
+
+## What a deployment is
+
+A _deployment_ encompasses the manifests, artifacts, configuration, and actions that deliver your code to remote environments. You can configure a deployment to deliver your [AWS Lambda](https://docs.aws.amazon.com/lambda) function to a single environment or multiple environments, either in sequence or in parallel depending on your [deployment configuration]({{<ref "deployment/create-deploy-config" >}}).
```

Comment:
let's not use 'manifests' for lambda. that is a kubernetes specific term, the generic term is 'artifacts'. (just remove 'the manifests, '

Reply:
ooops copy/paste and not updated